test/ui-test/testSuites/suite_communities/shared/scripts/community_names.py

```diff
@@ -85,6 +85,12 @@
 communitySettings_ColorPanel_HexColor_Input = {"container": statusDesktop_mainWindow_overlay, "objectName": "communityColorPanelHexInput", "type": "TextEdit",  "visible": True}
 communitySettings_SaveColor_Button = {"container": statusDesktop_mainWindow_overlay, "objectName": "communityColorPanelSelectColorButton", "type": "StatusButton", "visible": True}
 
+# Community transfer ownership
+copyCommunityPrivateKey = {"container": statusDesktop_mainWindow, "objectName": "copyCommunityPrivateKey", "type": "StatusButton", "visible": True}
+
+# Community Overview
+mainWindow_Back_up_StatusButton = {"container": statusDesktop_mainWindow, "objectName": "communityBannerButton", "occurrence": 3, "type": "StatusButton", "visible": True}
```

Comment:
Do we really need the `occurrence` here to determine this is the object?

Reply:
@noeliaSD there are 3 buttons with the same objectName belonging to StatusBanner parent, i did not create anything more simple rather than that. In case u know the better option - please let me know and i will fix it!

Reply:
Maybe adding an `objectName` into the parent `CommunityBanner`, adding the banner as an object in the `names.py` corresponding file and changing the `container` object in the button!

Reply:
this is fixed by your suggestion to find an object using a parent as a container